components/tikv_util/src/lib.rs

```diff
@@ -353,6 +352,15 @@ impl<L, R> Either<L, R> {
     }
 }
 
+impl<L, R> From<FutureEither<L, R>> for Either<L, R> {
```

Comment:
Where is it used?

Reply:
Nowhere to use it temporarily, I'll remove it later.